interactions/ext/console.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+
+import aiomonitor as aiomonitor
+
+import interactions
+from interactions import Extension
+
+__all__ = ("Console",)
+
+
+class Console(Extension):
+    """
+    Extension that starts the bot with the aiomonitor console active - notably giving you REPL for the bot
+
+    To access the console, you need to connect to the port specified in the constructor, by default 501.
+    On linux, you can do this with `nc localhost 501`, on windows you can use `telnet localhost 501`.
+
+    For both platforms you can also use "python -m aiomonitor.cli -p 501" as a replacement for the above commands.
+
+    Args:
+        port: The port to start the aiomonitor on
+        console_port: The port to start the aiomonitor console on
+        **kwargs: The locals to make available in the console, by default this includes `client`, `bot` and `interactions`
+
+    """
+
+    def __init__(self, client: interactions.Client, port: int = 501, console_port: int = 502, **kwargs) -> None:
```

Comment:
Can you add a `setup` function that exposes these parameters?

Reply:
Why? The setup hook isn't required to pass arguments to an extension?

`bot.load_extension("interactions.ext.console", port=4000)` works fine without a setup hook

Reply:
Oh, I must have missed that. Neat